rosbag2_converter_default_plugins/src/rosbag2_converter_default_plugins/cdr/cdr_converter.cpp

```diff
@@ -63,37 +63,56 @@ std::string get_package_library_path(const std::string & package_name)
 CdrConverter::CdrConverter()
 {
   auto library_path = get_package_library_path("rmw_fastrtps_cpp");
-  std::shared_ptr<Poco::SharedLibrary> library;
-  try {
-    library = std::make_shared<Poco::SharedLibrary>(library_path);
-  } catch (Poco::LibraryLoadException &) {
-    throw std::runtime_error(
-            std::string("poco exception: library could not be found:") + library_path);
+  rcutils_shared_library_t * library = nullptr;
+
+  library = new rcutils_shared_library_t;
+  *library = rcutils_get_zero_initialized_shared_library();
+
+  rcutils_ret_t ret = rcutils_load_shared_library(library, library_path.c_str());
+  if (ret != RCUTILS_RET_OK) {
+    delete library;
+    if (ret == RCUTILS_RET_BAD_ALLOC) {
+      throw std::runtime_error(
+              std::string("rcutils shared_library exception: failed to allocate memory"));
```

Comment:
```suggestion
      throw std::runtime_error{"rcutils shared_library exception: failed to allocate memory"};
```
ditto below.

Reply:
[Check this out](https://blog.quasardb.net/2017/03/05/cpp-braced-initialization)